target/linux/lantiq/base-files/etc/board.d/02_network

```diff
@@ -152,7 +152,20 @@ WBMR300)
 	ucidef_add_switch "switch0" \
 		"2:lan:1" "3:lan:2" "5:lan:3" "4:wan:1" "6t@eth0"
 	;;
-
+RS353)
+        ucidef_add_switch "switch0" \
+                "4:lan:1" "2:lan:2" "1:lan:3" "0:lan:4" "5:lan:5" "6t@eth0"
+	# Little hack to get the macaddr from flash and reorder bytes
```

Comment:
1. Sorry for that, fixed
2. The byte order between the original f/w and openrt (or U-boot) is not the same. So I have to reorder bytes. But this not an elegant way.
3. OK.